notebooks/407-person-tracking-webcam/407-person-tracking.ipynb

```diff
@@ -0,0 +1,616 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Person Tracking\n",
+    "\n",
+    "This notebook shows a person tracking scenario: it reads frames from an input video sequence, detects people in the frames, uniquely identify each one of them and track all of them until they leave the frame. We use the [Deep SORT](https://arxiv.org/abs/1703.07402) algorithm to perform object tracking, an extension to SORT (Simple Real time Tracker).\n",
+    "\n",
+    "## Detection vs Tracking\n",
+    "- In object detection, we detect an object in a frame, put a bounding box or a mask around it and classify the object. Note that, the job of the detector ends here. It processes each frame independently and identifies numerous objects in that particular frame. \n",
+    "- An object tracker on the other hand needs to track a particular object across the entire video. If the detector detects 3 cars in the frame, the object tracker has to identify the 3 separate detections and needs to track it across the subsequent frames (with the help of a unique ID).\n",
+    "\n",
+    "## Deep SORT\n",
+    "DeepSORT can be defined as the tracking algorithm which tracks objects not only based on the velocity and motion of the object but also the appearance of the object.\n",
```

Comment:
can you please provide link on article, where deep sort algo is described here too? and also for other terminology which is behind of scene (Kalman filter, Mahalanobis distance, Hungarian algorithm e.t.c.)

Reply:
done